bugbot/rules/not_landed.py

```diff
@@ -354,6 +371,11 @@ def get_bugs(self, date="today", bug_ids=[]):
             if not assignee:
                 continue
 
+            dependencies = data.get("dependencies", [])
+
+            if dependencies:
+                continue
```

Comment:
With this, the bug will still be shown in the final results as it has already been added to the `res` dictionary.

Similar to other cases, this should be filtered out in the `get_patch_data()` method.

Reply:
Done in 537574f845a107c9985aa4ba5439c34a6f3de55c, I'm now filtering it out in the `get_patch_data()` method.